layouts/shortcodes/subsections-list.html

```diff
@@ -2,7 +2,9 @@
 <div class="section-index subsection-list">
     <div class="entry">
         {{ range (.Page.Pages).Reverse  }}
-            <h5><a href="{{ .RelPermalink }}">{{ .LinkTitle }}</a></h5>
+            {{- if (ne .Params.no_version_warning true) -}}
```

Comment:
Is there a `not` operator to avoid explicit comparison to true?

Reply:
There is [`isset`](https://gohugo.io/functions/isset/), but I think that makes it _more_ complicated, as you could set `no_version_warning = false`, etc.

FWIW, I am not a fan of how go templates are abused in Hugo at large. But here we are.